packages/terra-application-name/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "terra-application-name",
-  "main": "lib/ApplicationName.js",
+  "main": "lib/ApplicationHeaderName.js",
```

Comment:
This package contains the application header name and application menu name. I think your previous package name was correct. We should also keep this in sync with the package name in lerna to avoid confusion (in most cases).